### ISSUE TITLE
Character sheets for cyberdecks, fortresses and programs

### ISSUE THREAD

**Issue by TheWing** [OctarineSourcerer/cyberpunk2020#91]
Hi.

I've been figuring out ways to run netrunning stuff with the current system, and it seems to be quite well suited for it even in its current state. Here's some of the things I've so far come to yearn for however, in place of just using NPC sheets for tokens on netspace maps:

### Program sheets
I've so far been using NPC sheets with the program information/description in Life-tab, and a Strength Feat -skill for rolling STR checks against code gates and other programs. This could be simplified a lot with a specific Program-sheet, which would only require information as follows:
1. Program STR value, a similar UI element to the character stats (INT/REF/so on)
2. Program MU size, just a number value, in the future it could be used to keep track of Memory Space left on a cyberdeck or other system
3. Program Cost, in eb. Nothing fancy here :)
4. Description tab, basically the same as Life-tab in character sheets
5. In case of Daemon programs, a field for how many programs it can carry?

### Fortress/Cyberdeck sheet
A sheet basically detailed on p.135 in the Cyberpunk 2020 source book;
1. Name
2. Number of CPU
3. Type (Cellular, Portable, Combat, Cyberlimb, Standard in case of cyberdecks, not sure about fortress?)
4. Cost
5. Data wall Strength
6. Code gate strengths (these are per gate, so not sure if they need to be in the sheet. I've been just denoting these straight on the battlemap)
7. Speed
8. Memory
9. List of programs

### AI Sheet
In case a Netspace fortress has INT > 12 (CPU>3), it houses an AI, and so far the NPC character sheet has been pretty good for this. AI just has no need for Combat, Gear or Cyberware, they seem to only require Skills, Life and some basic stats?

**Comment by TaladPS** [comment]
Check my pull request on cyberdecks and programs, I started on it.

**Comment by OctarineSourcerer** [comment]
Yep cyberdecks and program sheets (no interactions yet) are now in the `cyberdecks` branch. There's still a bit to go before I can put them in, and those are tracked at issues #99, #100, and #101 respectively.
